src/class_objectProcessor.py

```diff
@@ -765,10 +777,10 @@ def processmsg(self, data):
         # Don't send ACK if invalid, blacklisted senders, invisible
         # messages, disabled or chan
         if (
-            self.ackDataHasAValidHeader(ackData) and not blockMessage and
-            messageEncodingType != 0 and
-            not BMConfigParser().safeGetBoolean(toAddress, 'dontsendack') and
-            not BMConfigParser().safeGetBoolean(toAddress, 'chan')
+                self.ackDataHasAValidHeader(ackData) and not blockMessage and
+                messageEncodingType != 0 and
+                not BMConfigParser().safeGetBoolean(toAddress, 'dontsendack') and
+                not BMConfigParser().safeGetBoolean(toAddress, 'chan')
```

Comment:
Oh, I tired to point you to this pylint bug...

Reply:
oh, yes! Thank you . I'll fix!